src/test/components/__snapshots__/CallNodeContextMenu.test.js.snap

```diff
@@ -1,1184 +1,234 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
To look at these diffs, it's a lot better to configure your "diff settings" above to `hide whitespace changes`.

Then you'll see that what disappears is only the React components and their props. I find them very distracting when looking at a change, as sometimes we can change some component name or architecture but with the same output.